datafusion/core/src/execution/context/mod.rs

```diff
@@ -495,6 +496,32 @@ impl SessionContext {
                 self.set_variable(stmt).await
             }
 
+            LogicalPlan::Statement(Statement::CreateFunction(stmt)) => {
```

Comment:
Since CreateFunction / DropFunction modify the catalog, what do you think about making them part of `DdlStatement` instead of a `Statement`?